src/TwigComponent/doc/index.rst

```diff
@@ -840,86 +841,96 @@ The ``with`` data is what's mounted on the component object.
 .. note::
 
     Embedded components *cannot* currently be used with LiveComponents.
-    
+
 Component HTML Syntax
 ---------------------
-    
+
 .. versionadded:: 2.8
 
     This syntax was been introduced in 2.8 and is still experimental: it may change in the future.
-    
+
 Twig Components come with an HTML-like syntax to ease the readability of your template:
 
 .. code-block:: html+twig
 
-    <twig:Alert></:Alert>
+    <twig:Alert></twig:Alert>
     // or use a self-closing tag
-    <twig:Alert/>
-    
-You can pass props to your component by using HTML attributes. Suppose you have the following component:
+    <twig:Alert />
+
+Passing Props as HTML Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Passing props is done with HTML attributes. For example if you have this component::
+
+    #[AsTwigComponent]
+    class Alert
+    {
+        public string $message = '';
+        public bool $withActions = false;
+        public string $type = 'success';
+    }
+
+You can pass the ``message``, ``withActions`` or ``type`` props as attributes:
 
 .. code-block:: html+twig
 
-    // "withActions" property will be set to true
-    <twig:Alert withActions message="hello"></:Alert>
-        
-You can add the ':' prefix to your attribute to indicate that the value 
-should be compiled by Twig
+    // withActions will be set to true
+    <twig:Alert type="info" message="hello!" withActions />
+
+To pass in a dynamic value, prefix the attribute with ``:`` or use the
```

Comment:
Just asking ... is this convention based on the `:` prefix imposed somehow to us? Otherwise, it looks strange to introduce this new convention when we all know and use the `{{ ... }}` to output the value of variables or expressions. Thanks.

Reply:
It's borrowed from Vue, and also is used [Blade Components](https://laravel.com/docs/10.x/blade#component-attributes). I agree that it feels like an "extra" thing, but it's also a very common syntax to see.

Reply:
And you can still use the `{{ ... }}`, the `:` is just a small shortcut.

Reply:
OK, so this is a very common syntax outside of Symfony. Then, it makes sense to have it too. Thanks.

Reply:
Btw, I found that it is more or less "needed" when you want to pass non-string values - like booleans or arrays:

```html
<twig:Alert :showClose="false" :messages="['hello', 'world']" />
```